### DISCUSSION TITLE
Nicotine+ seems to stop downloading in the background

### DISCUSSION THREAD

**Discussion by urdaddyVlex** [nicotine-plus/nicotine-plus#1997] [Q&A]
I'm using Nicotine+ 3.2.2. It seems to stop downloading files minutes after I minimize the window. I tried to set "Toggle away status after minutes of inactivity" to 10000 and it did not work. Modifying the regedit doesn't work too.
I'm considering to use gpedit.msc and force allow Nicotine+ to run in the background, but I dunno how to get its package family name since it's not a Microsoft App. Any soulution?

**Comment by slook** [comment]
The Away/Online status has nothing to do with keeping the connection active, you can be "Away" whilst downloading/uploading this is only to indicate your attendance to users in Chat Rooms.

The application executable is `Nicotine+.exe` the package name is something like `nicotine` or `nicotine-plus` , but there should not be any specific group policy that is needed to maintain a stable connection.

It seems as if you have some other software on your computer interrupting the connection, perhaps this is to do with Power Saving (is the machine configured as a laptop?) or maybe some AV/security package, that requires configuring or uninstalling.

You can try enabling the "Connections" logging (Show Log History pane -> right click menu -> Log Categories -> "Connections" -> On) to see what errors happen whilst the application is minimized.

**Reply by urdaddyVlex** [reply]
Yeah I'm using a laptop (power is always connected tho) and also using a VPN otherwise download speed will be super slow. I've enabled Log History Pane and will wait till it automatically log off. Will submit my log history later. Thank ya

**Reply by urdaddyVlex** [reply]
Here's my log
02:21:31 Loading GTK 3.24.33
02:21:31 Loading Python 3.9.10 (main, Mar 14 2022, 18:37:07)  [GCC 11.2.0 64 bit (AMD64)]
02:21:31 Loading Nicotine+ 3.2.2
02:21:31 Listening on port: 2234
02:21:31 Loading plugin system
02:21:31 Connecting to server.slsknet.org:2242
02:21:31 Rescanning shares…
02:21:31 761 folders found before rescan, rebuilding…
02:21:31 Connected to server server.slsknet.org:2242, logging in…
02:21:32 UPnP: Failed to forward external port 2234: UPnP is not available on this network
02:21:32 1804 privileged users
02:21:34 31 days, 10 hours, 38 minutes, 35 seconds of download privileges left.
02:21:34 User 4010 is online
02:21:40 Rescan complete: 768 folders found

Nicotine+ says download was cancelled. Does this mean it's because of my firewall?
Btw if I click resume download won't begin. I have to restart the app

**Reply by slook** [reply]
I'm not sure the log isn't detailed enough to show the problem.

Please enable from the right-click context menu of the log history pane -> Log Catagories -> "[Debug] Connections".

**Reply by urdaddyVlex** [reply]
https://1drv.ms/t/s!AtJyYhh7zgsUjD8nc3WgQMsmTm-K?e=2YkROF
Here's my full log since last start. Hope this will be helpful

**Reply by slook** [reply]
`[Conn] Sending message of type <class 'pynicotine.slskmessages.QueueUpload'> to user`
`[Conn] Sending message of type <class 'pynicotine.slskmessages.PlaceInQueueRequest'> to user`

There are lots of these repeated in quick succession, perhaps this causes some behaviour which could lead to a temporary ban, after asking to place a queued transfer many times obviously there is some sort of error with one of the items in Downloads.
